static/js/components/GroupSources.jsx

```diff
@@ -0,0 +1,250 @@
+import React, { useEffect, Suspense } from "react";
+import PropTypes from "prop-types";
+import { useSelector, useDispatch } from "react-redux";
+
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import Grid from "@material-ui/core/Grid";
+// import BuildIcon from "@material-ui/icons/Build";
+
+import Link from "@material-ui/core/Link";
+import PictureAsPdfIcon from "@material-ui/icons/PictureAsPdf";
+
+// import Menu from "@material-ui/core/Menu";
+// import MenuItem from "@material-ui/core/MenuItem";
+
+import MUIDataTable from "mui-datatables";
+import ThumbnailList from "./ThumbnailList";
+import styles from "./RunSummary.css";
+// import { observingRunTitle } from "./AssignmentForm";
+// import { ObservingRunStarList } from "./StarList";
+import * as SourcesAction from "../ducks/sources";
+// import * as Action from "../ducks/observingRun";
+import { ra_to_hours, dec_to_hours } from "../units";
+
+const VegaPlot = React.lazy(() => import("./VegaPlot"));
+
+const GroupSources = ({ route }) => {
+  const dispatch = useDispatch();
+  const sources = useSelector((state) => state.sources.latest);
+  const groups = useSelector((state) => state.groups.user);
+
+  // Load the group sources
+  useEffect(() => {
+    dispatch(SourcesAction.fetchSources({ group_id: route.id }));
+  }, [route.id, dispatch]);
+
+  if (sources === undefined || sources === null) {
+    return "Loading Sources...";
+  }
+
+  const group_id = parseInt(route.id, 10);
+  let group_name = "";
+
+  if (groups === undefined || groups === null) {
+    // nothing, just wait
+  } else {
+    const group_obj = groups.find((obj) => {
```

Comment:
```suggestion
    const group = groups.find((group) => {
```
This is very confusing variable naming. Obj is a meaningful variable name, don't use that for a group instance.

Reply:
Yes, this is true in astronomy in general when doing any OOP and looking at celestial objects. I'll change it.